modules/dynatrace-private-activegate/cloudconfig.tpl

```diff
@@ -0,0 +1,14 @@
+#cloud-config
+package_upgrade: true
+
+runcmd:
+ - apt-get update
+ - apt-get -y install xvfb x11-xkb-utils xfonts-100dpi xfonts-75dpi xfonts-scalable
+ - apt-get -y install libasound2 libatk-bridge2.0-0 libatk1.0-0 libc6:amd64 libcairo2 libcups2 libgdk-pixbuf2.0-0 libgtk-3-0 libnspr4 libnss3 libxss1 xdg-utils
+ - curl --output chromium.tgz https://synthetic-packages.s3.amazonaws.com/Chromium/deb/chromium-89.0.4389.82-0ubuntu0.18.04.1.tgz
```

Comment:
should probably be a variable in the module?

Reply:
Will create variable if I go with the 1 module approach for both set of active gates